resources/config.yml

```diff
@@ -25,11 +25,12 @@
 # !Packs must be added to ./plugins/ProtocolSupportPocketStuff/pocketpacks!
 #
 # = Hacks =
-# dimensions  - Enables using bukkit to fake dimension switching for pocket players.
-# holograms   - Enables experimental caching and "armor stand to player" replacement to support holograms for PE players.
-# itemframes  - Enables experimental caching and mucking with packets to support itemframes for PE players.
-# teams       - Enables experimental caching and mucking with packets to support team prefix/suffixes for PE players.
-# bossbars    - Enables experimental caching and mucking with packets to boss bars for PE players.
+# dimensions          - Enables using bukkit to fake dimension switching for pocket players.
+# holograms           - Enables experimental caching and "armor stand to player" replacement to support holograms for PE players.
+# itemframes          - Enables experimental caching and mucking with packets to support itemframes for PE players.
+# teams               - Enables experimental caching and mucking with packets to support team prefix/suffixes for PE players.
+# bossbars            - Enables experimental caching and mucking with packets to boss bars for PE players.
+# player-heads-skins  - Enables experimental caching, "player heads to players" replacement and skin replacement to support player heads with skins for PE players.
```

Comment:
Could you make two seperate options for this? One for faking skull with player, the other for mucking with player skins.